assets/js/blocks/checkout/inner-blocks/checkout-billing-address-block/attributes.tsx

```diff
@@ -1,20 +1,13 @@
-/**
- * External dependencies
- */
-import { __ } from '@wordpress/i18n';
-
 /**
  * Internal dependencies
  */
 import formStepAttributes from '../../form-step/attributes';
+import { DEFAULT_TITLE, DEFAULT_DESCRIPTION } from './constants';
 
-export default {
+export const attributes: Record< string, Record< string, unknown > > = {
```

Comment:
Could we use the Gutenberg type `BlockAttributes` here?

You'd need to `import { BlockAttributes } from '@wordpress/blocks';`

Reply:
Yes, that's a better way to define types. Added the `BlockAttributes` type. 👍